src/display/annotation_layer.js

```diff
@@ -1615,7 +1622,12 @@ class AnnotationLayer {
             parameters.div.appendChild(renderedElement);
           }
         } else {
-          parameters.div.appendChild(rendered);
+          if (element.popup) {
```

Comment:
*Drive-by-comment:* I don't think you need to add a boolean for this, since `element instanceof PopupAnnotationElement` ought to work just as well :-)